src/main/java/com/sigma/cinematicketpro/exception/UserAlreadyExistsException.java

```diff
@@ -0,0 +1,15 @@
+package com.sigma.cinematicketpro.exception;
+
+import lombok.Getter;
+
+import java.util.List;
+
+public class UserAlreadyExistsException extends RuntimeException {
```

Comment:
I am pretty sure that there is smth in spring security for this

Reply:
Seems like the only similar exception is org.springframework.security.acls.model.AlreadyExistsException. But it was either removed in spring boot 3.x.x or requires another dependency